examples/tutorials/aposmm/aposmm_tutorial_notebook.ipynb

```diff
@@ -33,11 +33,11 @@
     "def six_hump_camel(H, persis_info, sim_specs, _):\n",
     "    \"\"\"Six-Hump Camel sim_f.\"\"\"\n",
     "\n",
-    "    batch = len(H['x'])  # Num evaluations each sim_f call.\n",
-    "    H_o = np.zeros(batch, dtype=sim_specs['out'])  # Define output array H\n",
+    "    batch = len(H[\"x\"])  # Num evaluations each sim_f call.\n",
```

Comment:
@jlnav I assume that this is correctly formatted Jupyter, right?

Reply:
Seems so, the code looks normal when the notebook is run locally